docs/topics/downloader-middleware.rst

```diff
@@ -1020,10 +1020,10 @@ the request will be ignored by this middleware even if
 RobotFileParser
 ~~~~~~~~~~~~~~~
 
-`RobotFileParser <https://docs.python.org/3.7/library/urllib.robotparser.html>`_ is 
-Python's inbuilt robots.txt_ parser. The parser is fully compliant with `Martijn Koster's 
+`RobotFileParser <https://docs.python.org/3.7/library/urllib.robotparser.html>`_ is
+Python's inbuilt robots.txt_ parser. The parser is fully compliant with `Martijn Koster's
```

Comment:
```suggestion
Python's built-in robots.txt_ parser. The parser is fully compliant with `Martijn Koster's
```
I know this line was like this before, but since it's being changed we might as well fix it.